index.js

```diff
@@ -15,6 +126,7 @@ menuClose.addEventListener('click', () => {
   navMenu.classList.toggle('show-mobile-nav');
 });
 
+// hides the nav drawer when a nav item is clicked
 for (let i = 0; i < linkItems.length; i += 1) {
   linkItems[i].addEventListener('click', () => {
     navMenu.classList.remove('show-mobile-nav');
```

Comment:
- [ ] The popup should extend all the way through the page:

Desing:
![image](https://user-images.githubusercontent.com/62705947/195371657-be57b24b-ef9f-4522-a897-6aa253056d81.png)
Yours:
![image](https://user-images.githubusercontent.com/62705947/195371775-b51918e8-dacb-4369-81d0-a71d21c5ced4.png)

Reply:
- [ ] Check the font color and adjust it as on the design, it should be: #3A4A42;

Reply:
- [ ] Move these elements more up in order to match better the design

Yours:
![image](https://user-images.githubusercontent.com/62705947/195372881-8f308645-8b89-4211-aa49-527f1050b613.png)
Design: 
![image](https://user-images.githubusercontent.com/62705947/195372955-e142f214-59d0-4eb5-b7d7-5db2389ea6b8.png)